Gemfile

```diff
@@ -0,0 +1,41 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.3'
```

Comment:
Are you sure about this version? Will you be able to run CI on Semaphore?

Reply:
Changed it to 2.5.1